engines/director/cursor.cpp

```diff
@@ -82,7 +82,7 @@ void Cursor::readFromCast(Datum cursorCasts) {
 	_usePalette = false;
 	_keyColor = 3;
 
-	resetCursor(Graphics::kMacCursorCustom, true, cursorCasts);
+	resetCursor(Graphics::kMacCursorCustom, true, Datum(cursorId.member));
```

Comment:
Hmm. What if the cursor is defined/referenced from another cast in D5+?

Reply:
I mean, that in this case cursor is referenced by `cursorId.cast` + `cursorId.member`

Reply:
?

Reply:
Fixed

Reply:
(tested in D4 and D5)